WcaOnRails/app/assets/stylesheets/collapsible-panels.scss

```diff
@@ -1,9 +1,9 @@
 .collapse-indicator {
-  @extend .glyphicon;
-  @extend .glyphicon-chevron-down;
+  @extend .icon;
+  content: "\f078"; // Semantic chevron down
 
   .collapsed & {
-    @extend .glyphicon-chevron-right;
```

Comment:
It used to be `chevron-right`, but you're picking "chevron up" as a replacement.
I think it should be `f054` for "chevron right": https://github.com/fomantic/Fomantic-UI-SASS/blob/master/app/assets/stylesheets/semantic-ui/elements/_icon.scss#L1299

Reply:
not sure why you linked `i.icon.cloud.meatball` but I changed it ;) to [chevron right](https://github.com/fomantic/Fomantic-UI-SASS/blob/87a7b23c5d6a598043d13f32a2e9875d6b8f80f2/app/assets/stylesheets/semantic-ui/elements/_icon.scss#L1252)